### PR TITLE
refactor: VDRI: remove create for http binding

### LINKED ISSUE #1819: VDRI: remove create for http binding

### PR DESCRIPTION
**Pull request by fqutishat**
- Update to latest sidetree-mock
- Create DID outside aries

closes #1819

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Comment by fqutishat** [comment]
@troyronda @Baha-sk @sandrask on hold until we use new kms for pack logic

File: test/aries-js-worker/fixtures/.env [hunk 1]
```diff
@@ -37,11 +37,11 @@ E2E_USER_API_PORT=10082
...
-HTTP_LOCAL_DID_RESOLVER=sidetree@http://localhost:48326/document
+HTTP_DID_RESOLVER=sidetree@http://sidetree-jsmock:48326/sidetree/0.0.1/identifiers
+HTTP_LOCAL_RESOLVER_URL=http://localhost:48326/sidetree/0.0.1/identifiers
+HTTP_LOCAL_DID_RESOLVER=sidetree@http://localhost:48326/sidetree/0.0.1/identifiers
 
 # sidetree mock configurations
 COMPOSE_DIR=.
-SIDETREE_MOCK_FIXTURE_IMAGE=docker.pkg.github.com/trustbloc/sidetree-mock/sidetree-mock
-SIDETREE_MOCK_FIXTURE_IMAGE_TAG=0.1.2
+SIDETREE_MOCK_FIXTURE_IMAGE=docker.pkg.github.com/trustbloc-cicd/snapshot/sidetree-mock
```
Comment (troyronda) [inline test/aries-js-worker/fixtures/.env:46]: why not release version?

File: test/aries-js-worker/test/vdri/vdri.js [hunk 2]
```diff
@@ -54,62 +37,66 @@ describe("VDRI", function () {
...
-                if (didID === resp.result[j].id){
-                    assert.equal(didName, resp.result[j].name)
-                    return
-                }
-            }
-        }
-
-        throw new Error("Alice doesn't have did")
-    })
+    // it("Alice retrieves the did from store",  async function () {
```
Comment (troyronda) [inline test/aries-js-worker/test/vdri/vdri.js:66]: why commented out

File: test/aries-js-worker/fixtures/.env (hunk 1 above)
Comment (fqutishat) [reply test/aries-js-worker/fixtures/.env:46]: we need this fix https://github.com/trustbloc/sidetree-core-go/commit/2f2b686a83c387731f70b86ef41b04663477b1f7 after release

File: test/aries-js-worker/test/vdri/vdri.js (hunk 2 above)
Comment (fqutishat) [reply test/aries-js-worker/test/vdri/vdri.js]: I commented out by mistake. is fixed now

File: test/aries-js-worker/test/vdri/vdri.js [hunk 3]
```diff
@@ -94,7 +81,7 @@ describe("VDRI", function () {
     })
 
     it("Alice validates that she has the did", async function () {
-        for (let i = 0; i < agents.length; i++) {
+        for (let i = 0; i < 1; i++) {
```
Comment (sandrask) [inline test/aries-js-worker/test/vdri/vdri.js:84]: we have for loop for length of one in multiple places

**Pull request merged**